src/sentry/sentry_metrics/use_case_id_registry.py

```diff
@@ -7,14 +7,16 @@
 
 
 class UseCaseID(Enum):
+    SPANS = "spans"
     TRANSACTIONS = "transactions"
     SESSIONS = "sessions"
 
 
 # UseCaseKey will be renamed to MetricPathKey
 METRIC_PATH_MAPPING: Mapping[UseCaseID, UseCaseKey] = {
-    UseCaseID.SESSIONS: UseCaseKey.RELEASE_HEALTH,
+    UseCaseID.SPANS: UseCaseKey.PERFORMANCE,
```

Comment:
Since we added the spans use case here in order to test out the write limiter functionality, could we maybe use a mock/test UseCaseID instead of spans?

Reply:
Done